CollapseLauncher/XAMLs/MainApp/Pages/GameSettingsPages/StarRailGameSettingsPage.Ext.cs

```diff
@@ -222,7 +247,7 @@ public int LightQuality
         public int CharacterQuality
         {
             get => (int)Settings.GraphicsSettings.CharacterQuality;
-            set => Settings.GraphicsSettings.CharacterQuality = (Quality)value;
+            set => Settings.GraphicsSettings.CharacterQuality = (CharacterQuality)value;
```

Comment:
To avoid potential ``Value does not fall within the expected range.`` exception or any arbitrary selections if the value is more than 4 (High) or less than 2 (Low), we need to add such a fallback handler.

So, in this case for the getter:
1. If ``(int)CharacterQuality`` < 2, then return and force set the ``CharacterQuality`` to ``CharacterQuality.Low``.
2. If ``(int)CharacterQuality`` > 4, then return and force set the ``CharacterQuality`` to ``CharacterQuality.High``